### ISSUE TITLE
No snippets are working, even though they use to work before

### ISSUE THREAD

**Issue by drjosephliu** [SirVer/ultisnips#1220]
**Expected behavior:**
Expecting snippets to work.

**Actual behavior:**
None of them are working.

**Steps to reproduce**

For some reason UltiSnips stopped working since 2 weeks ago. I use it mainly for latex and have many snippets, one example of which is:

```
snippet beg "begin{} / end{}" bA
\\begin{$1}
	$0
\\end{$1}
endsnippet
```

And none of them work.  

My `.vimrc` settings:

let g:UltiSnipsSnippetDirectories=[$HOME.'/.config./nvim/UltiSnips']
let g:UltiSnipsExpandTrigger = '<c-w>'
let g:UltiSnipsJumpForwardTrigger = '<c-w>'
let g:UltiSnipsJumpBackwardTrigger = '<s-tab>'

In a `.tex` file I can also confirm the global variables are being found correctly by running
`:echo g:UltiSnipsSnippetDirectors`

Not sure what I can do to diagnose the problem. 

-----

- **Operating System**: 
- **Vim Version**: 
- **UltiSnips Version**: 
- **Python inside Vim**: 
- **Docker repo/vimrc**:

**Comment by akemrir** [comment]
OS: Archlinux
Vim: VIM - Vi IMproved 8.2  1-717
UltiSnips: master / tag 3.2
Python: 3

I have same problem. Snippets stopped expanding.
List of snippets in deoplete is visible, but on expanding they block Vim input.
I reverted to tag 3.2 and this don't help either.

**Comment by akemrir** [comment]
I`ve found issue in my terminal. Key combination which I am using to expand is bound to other function, that's why it stopped working. On newest version of ultisnips with python 3 it works.

**Comment by meredrica** [comment]
Expansion doesn't work for me either. YCM shows the snippet but when I select it, nothing happens.

**Comment by meredrica** [comment]
I just rolled all the way back to 303d8da0f492b1cc125ae13d0e810b2430622322, didn't help. Here's my system info:
OS: arch
Vim:
```
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled May 23 2020 22:02:57)
Included patches: 1-814
Compiled by Arch Linux
Huge version without GUI.  Features included (+) or not (-):
+acl               -farsi             +mouse_sgr         +tag_binary
+arabic            +file_in_path      -mouse_sysmouse    -tag_old_static
+autocmd           +find_in_path      +mouse_urxvt       -tag_any_white
+autochdir         +float             +mouse_xterm       +tcl/dyn
-autoservername    +folding           +multi_byte        +termguicolors
-balloon_eval      -footer            +multi_lang        +terminal
+balloon_eval_term +fork()            -mzscheme          +terminfo
-browse            +gettext           +netbeans_intg     +termresponse
++builtin_terms    -hangul_input      +num64             +textobjects
+byte_offset       +iconv             +packages          +textprop
+channel           +insert_expand     +path_extra        +timers
+cindent           +ipv6              +perl/dyn          +title
-clientserver      +job               +persistent_undo   -toolbar
-clipboard         +jumplist          +popupwin          +user_commands
+cmdline_compl     +keymap            +postscript        +vartabs
+cmdline_hist      +lambda            +printer           +vertsplit
+cmdline_info      +langmap           +profile           +virtualedit
+comments          +libcall           +python/dyn        +visual
+conceal           +linebreak         +python3/dyn       +visualextra
+cryptv            +lispindent        +quickfix          +viminfo
+cscope            +listcmds          +reltime           +vreplace
+cursorbind        +localmap          +rightleft         +wildignore
+cursorshape       +lua/dyn           +ruby/dyn          +wildmenu
+dialog_con        +menu              +scrollbind        +windows
+diff              +mksession         +signs             +writebackup
+digraphs          +modify_fname      +smartindent       -X11
-dnd               +mouse             -sound             -xfontset
-ebcdic            -mouseshape        +spell             -xim
+emacs_tags        +mouse_dec         +startuptime       -xpm
+eval              +mouse_gpm         +statusline        -xsmp
+ex_extra          -mouse_jsbterm     -sun_workshop      -xterm_clipboard
+extra_search      +mouse_netterm     +syntax            -xterm_save
   system vimrc file: "/etc/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H     -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       
Linking: gcc   -L. -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -fstack-protector-strong -rdynamic -Wl,-export-dynamic -Wl,-E -Wl,-rpath,/usr/lib/perl5/5.30/core_perl/CORE  -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -L/usr/local/lib -Wl,--as-needed -o vim        -lm -ltinfo -lelf    -lacl -lattr -lgpm -ldl   -Wl,-E -Wl,-rpath,/usr/lib/perl5/5.30/core_perl/CORE -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -fstack-protector-strong -L/usr/local/lib  -L/usr/lib/perl5/5.30/core_perl/CORE -lperl -lpthread -ldl -lm -lcrypt -lutil -lc   -L/usr/lib -ltclstub8.6 -ldl -lz -lpthread -lm     
```
UltiSnips: doesn't seem to matter.

**Comment by akemrir** [comment]
Hi there, check your python settings.
I`ve managed them to work with forcing vim to use python3. With this at very beggining of vimrc:
```
if has('python3')
endif
```

**Comment by meredrica** [comment]
Hi @akemrir I added that to the very top of my vimrc, doesn't change anything.
I'm sure vim runs py3 since I get the expected error: 
``E836: This Vim cannot execute :python after using :py3
E263: Sorry, this command is disabled, the Python library could not be loaded.
``

**Comment by meredrica** [comment]
Oh, I found this issue here: https://github.com/ycm-core/YouCompleteMe/issues/420
Basically what seems to happen is that there is a keybinding clash. There are some suggestions in the thread that I haven't checked out so far. I'll see if they work and post an update here.

**Comment by meredrica** [comment]
I can confirm that the keymap is the problem.
If you set something like `let g:UltiSnipsExpandTrigger="<leader>c"` the expansion works as expected.
A hint to this is actually already in the README but the wording is not the best.

**Comment by SirVer** [comment]
Closed by #1244
